libs/utils/src/lib/format/number.ts

```diff
@@ -90,7 +105,29 @@ export const formatNumber = (
   rawValue: string | number | BigNumber,
   formatDecimals = 0
 ) => {
-  return getNumberFormat(formatDecimals).format(Number(rawValue));
+  const decimalSeparator = getDecimalSeparator() || '.';
```

Comment:
I think the js docs for formatNumber and formatNumberFixed need updating, they both indicate the same thing.. we need to clarify